File: py/843.py
from python_modules.socket_ex import *
from python_configs.server_address import *

server = create_server(address_server_843)

packet = b'<cross-domain-policy><allow-access-from domain="*" to-ports="*" /></cross-domain-policy>'

while True:
	client, address = server.accept()
	client.recv(0x20)
	client.sendall(packet)
	client.close()

File: py/843.js
"use strict";

const serverPort = require("./node_configs/serverPort");
require("Socket");

const text = '<cross-domain-policy><allow-access-from domain="*" to-ports="*" /></cross-domain-policy>';
const packet = new Buffer(text);

require("net").createServer(socket => {
	socket.on("data", chunk => {
		socket.write(packet, "utf8", () => {
			socket.end();
			socket.removeAllListeners();
		});
	});
	socket.listenCloseEvent(10000, null);
}).listen(843, serverPort.gate_host);